README.md

```diff
@@ -128,6 +128,7 @@
 - [Import WebNN Experimental Feature](https://onnxruntime.ai/docs/get-started/with-javascript/web.html)
 - [Build with WebNN support via JavaScript Execution Provider](https://onnxruntime.ai/docs/build/web.html)
 - [WebNN Execution Provider](https://onnxruntime.ai/docs/tutorials/web/env-flags-and-session-options.html)
```

Comment:
Suggest change to:

- [ONNXRuntime Web Tutorials](https://onnxruntime.ai/docs/tutorials/web/)
- [Build ONNXRuntime Web with WebNN Support](https://onnxruntime.ai/docs/build/web.html)
- [WebNN Operators Support Table](https://github.com/microsoft/onnxruntime/blob/main/js/web/docs/webnn-operators.md) - by Wanming Lin

Reply:
Fixed